internal/verifier/logging_setup.go

```diff
@@ -29,3 +29,15 @@ func getLogWriter(logPath string) io.Writer {
 
 	return zerolog.SyncWriter(writer)
 }
+
+func getLoggerAndWriter(logPath string) (*logger.Logger, io.Writer) {
+	writer := getLogWriter(logPath)
+
+	consoleWriter := zerolog.ConsoleWriter{
```

Comment:
I'm not sure what this is doing here.  It feels like we probably shouldn't do this unconditionally. If we're logging to a file, using the console writer would spew a bunch of color codes into the log, wouldn't it?

Reply:
This is moved code.

Reply:
This is just moved code.